spec/domain/export/tabular/people/sac_recipient_household_row_spec.rb

```diff
@@ -0,0 +1,137 @@
+# frozen_string_literal: true
+
+#  Copyright (c) 2024, Schweizer Alpen-Club. This file is part of
+#  hitobito_sac_cas and licensed under the Affero General Public License version 3
+#  or later. See the COPYING file at the top-level directory or at
+#  https://github.com/hitobito/hitobito_sac_cas
+
+require "spec_helper"
+
+describe Export::Tabular::People::SacRecipientHouseholdRow do
+  let(:group) { groups(:bluemlisalp_mitglieder) }
+  let(:person) do
+    Fabricate.build(
+      :person,
+      id: 42,
+      first_name: "Hans",
+      last_name: "Muster",
+      street: "Musterstrasse",
+      housenumber: "42",
+      zip_code: "4242",
+      town: "Musterhausen",
+      country: "Schweiz",
+      email: "hans.muster@example.com"
+    )
+  end
+  let(:family_member) do
```

Comment:
es gibt bereits familienmitglied in den fixtures, das kann hier nicht verwendet werden, oder?

Reply:
Ich mags fast mehr dass alle Felder explizit im Spec file genannt werden weil ich die ja dann alle `expect`e
Zudem muss ich je nachdem sowieso den household_key anpassen und so, find ich dann angenehm wenn ich nicht an einer fixture Person rumeditiere